tests/schedulers/airflow/test_end_to_end.py

```diff
@@ -38,7 +38,9 @@ def test_sushi(mocker: MockerFixture, is_docker: bool):
         auto_apply=True,
     )
 
-    data_validator.validate("sushi.customer_revenue_lifetime", start, end)
+    data_validator.validate(
+        "sushi.customer_revenue_lifetime", start, yesterday(), env_name="test_dev"
```

Comment:
So it appears that when we pass in `now()` as end to plan then it gets converted into an inclusive yesterday. This function wants an inclusive start and end so I pass yesterday into it. The reason why it is inclusive is because it currently just handles dates and therefore you call it like you would a between.